controller/token.go

```diff
@@ -280,6 +284,18 @@ func (c *TokenController) Delete(ctx *app.DeleteTokenContext) error {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
 
+	// Delete from Keycloak
+	err = c.keycloakExternalTokenService.Delete(ctx, c.getKeycloakIdentityProviderURL(currentIdentity.String(), providerConfig.TypeName()))
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"error":         err,
+			"provider_name": providerConfig.TypeName(),
+			"identity_id":   currentIdentity,
+		}, "Unable to remove Identity Provider link from Keycloak.")
+		// Not critical. Log the error and proceed.
```

Comment:
:+1:  When this happens, users are gonna see that their account is still connected as long as we check keycloak first.